cmd/cluster_phase_etcd.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/xetys/hetzner-kube/cmd/phases"
+	"github.com/xetys/hetzner-kube/pkg"
+)
+
+func init() {
+	command := declarePhaseCommand("etcd", "setups a etcd cluster", func(cmd *cobra.Command, args []string) {
+		provider, clusterManager, coordinator := getCommonPhaseDependencies(6, cmd, args)
```

Comment:
Using 6 here is really confusing, I suggest to move into a const list (maybe iota?) to define a more comprehensible name using a liked list